machine.go

```diff
@@ -613,10 +613,16 @@ func (m *Machine) startVMM(ctx context.Context) error {
 		return err
 	}
 
-	// This goroutine is used to kill the process by context cancelletion,
+	// This goroutine is used to kill the process by context cancellation,
 	// but doesn't tell anyone about that.
 	go func() {
-		<-ctx.Done()
+		select {
+		case <-ctx.Done():
+			break
+		case <-m.cleanupCh:
```

Comment:
nitpick: `m.cleanupCh` works, but to to make it clear that what we really care about is whether the VM exited, can you use `m.exitCh`?

Reply:
Good call; done